reporter-web-app/src/models/WebAppPackage.js

```diff
@@ -139,6 +141,10 @@ class WebAppPackage {
                 this.#curations = obj.curations;
             }
 
+            if (obj.authors) {
```

Comment:
If statements are alphabetically sorted so this if statement should be in front of `if (obj.binary_artifact || obj.binaryArtifact) {`